VERSION

```diff
@@ -4,6 +4,8 @@
 6
 === [Zilliqa]Fix Version ===
 0
+=== [Zilliqa]rc version ===
+0
```

Comment:
@yaron-zilliqa Can you take a look at this one? Do we need to update the stuff in `docker/Makefile` to respect this?

Reply:
Don't recall these were ever respected. But basically it'd be good to, I suppose.

Reply:
Anyway, I think for now it can be ignored or removed.

Reply:
Looks like they are now: https://github.com/Zilliqa/Zilliqa/blob/4e74fea415feac83a926a0d87cee22cea6c57c1c/docker/Makefile#L9-L11
(I can't remember if I changed this or not)